recipes/tbb/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# clang: error: invalid deployment target for -stdlib=libc++
+export MACOSX_DEPLOYMENT_TARGET=10.7
```

Comment:
@jschueller you may have seen an error on your computer because you did not have the older SDK.  If you want the SDK to be able to build older stuff, you can get them at https://github.com/phracker/MacOSX-SDKs